four-node-cluster/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    # Copyright 2019 WSO2 Inc. (http://wso2.org)
```

Comment:
Licence

Reply:
Fixed by https://github.com/wso2/performance-is/pull/128/commits/6f54cb9f8218f1e659a5822fdc7b810124ba65f4